capstone/src/main/java/com/google/sps/data/Lesson.java

```diff
@@ -39,8 +36,27 @@ public Lesson(String type, String title, String description) {
         this.entity.setProperty(Lesson.DATE_PROPERTY_KEY, new Date());
     }
 
-    public Entity getLessonEntity() {
-        return this.entity;
+    @SuppressWarnings("unchecked")
+    public void addTag(String tag) {
+        if (this.entity.getProperty(Lesson.TAGS_PROPERTY_KEY) == null) {
```

Comment:
This should be Room.TAGS_PROPERTY_KEY but we've changed it in a later PR. Feel free to do it here though.